test/e2e/place_statvar_ranking_test.go

```diff
@@ -198,8 +198,8 @@ func TestChartConfigRankings(t *testing.T) {
 			})
 
 			goldenFile := path.Join(goldenPath, c.goldenFile)
-			if generateGolden {
-				updateGolden(missingRankings, goldenPath, c.goldenFile)
+			if GenerateGolden {
```

Comment:
Were these other style updates automated? Which tool did you use to generate these? Would be nice to standardize on these so we can keep this in sync.

Reply:
Updated using vscode IDE.